packages/web-components/project.json

```diff
@@ -5,7 +5,6 @@
   "implicitDependencies": [],
   "tags": ["platform:web", "web-components"],
   "targets": {
-    "e2e": { "dependsOn": ["build-storybook"] },
-    "type-check": { "dependsOn": ["build"] }
```

Comment:
this makes running `build` twice on pipeline and block parallelisation. it doesn't make sense to run build  on `self` which runs `tsc` and then run same `tsc` again during type-check